### PR TITLE
Update swagger responses - Closes #224

### LINKED ISSUE #224: Update Swagger responses
### Description

The new API updates require manual updates for the Swagger-based specification response format.

Please update it accordingly so it reflects the recent SDK updates.

### Motivation

- Swagger specs is required by the UI developers to develop against the Service API
- New features need to have the API documentation updated

### Acceptance Criteria

- All endpoints described in the concepts and documentation are updated
- Swagger specs file is rendered properly by the Swagger UI

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #224 

### How was it solved?

- [x] All endpoints are updated
- [x] Swagger specs file is rendered properly by the Swagger UI

### How was it tested?
Local + Swagger editor

### CODE REVIEW

File: services/gateway/apis/http-version1/methods/transactionsStatisticsDay.js [hunk 1]
```diff
@@ -30,10 +30,11 @@ module.exports = {
 		const transactionSchema = {};
 		transactionSchema[this.swaggerApiPath] = { get: {} };
 		transactionSchema[this.swaggerApiPath].get.tags = this.tags;
+		transactionSchema[this.swaggerApiPath].get.summary = 'Requests transaction statistics day';
 		transactionSchema[this.swaggerApiPath].get.parameters = transformParams('transactions', this.params);
 		transactionSchema[this.swaggerApiPath].get.responses = {
 			200: {
-				description: 'array of transactions with details',
+				description: 'array of transactions statisctis day',
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version1/methods/transactionsStatisticsDay.js:37]:
```suggestion
				description: 'array of transactions statistics aggregated per day',
```

File: services/gateway/apis/http-version1/methods/transactionsStatisticsMonth.js [hunk 2]
```diff
@@ -30,10 +30,11 @@ module.exports = {
 		const transactionSchema = {};
 		transactionSchema[this.swaggerApiPath] = { get: {} };
 		transactionSchema[this.swaggerApiPath].get.tags = this.tags;
+		transactionSchema[this.swaggerApiPath].get.summary = 'Requests transaction statistics month';
 		transactionSchema[this.swaggerApiPath].get.parameters = transformParams('transactions', this.params);
 		transactionSchema[this.swaggerApiPath].get.responses = {
 			200: {
-				description: 'array of transactions with details',
+				description: 'array of transactions statistics month',
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version1/methods/transactionsStatisticsMonth.js:37]:
```suggestion
				description: 'array of transactions statistics aggregated per month',
```

**Review by sameersubudhi — APPROVED** [review]
Looks good overall! However, please take into consideration the suggested changes.

File: services/gateway/apis/http-version1/methods/transactionsStatisticsDay.js (hunk 1 above)
Comment (nagdahimanshu) [reply services/gateway/apis/http-version1/methods/transactionsStatisticsDay.js]: Done

File: services/gateway/apis/http-version1/methods/transactionsStatisticsMonth.js (hunk 2 above)
Comment (nagdahimanshu) [reply services/gateway/apis/http-version1/methods/transactionsStatisticsMonth.js]: Done

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**